cylc/flow/network/client.py

```diff
@@ -184,8 +184,10 @@ def serial_request(self, command, args=None, timeout=None):
                 nothing more, nothing less.
 
         """
-        return asyncio.run(
-            self.async_request(command, args, timeout))
+        loop = asyncio.get_event_loop()
+        task = loop.create_task(self.async_request(command, args, timeout))
+        result = loop.run_until_complete(task)
+        return task.result()
```

Comment:
The `result` variable is not being used. I think we can `return loop.run_until_complete(task)` instead?

Reply:
I think `loop.run_until_complete(task)` would return a boolean. Will remove the unused var.

Reply:
From the code comment in the IDE, it looked like it returned the result (same one set in the Task I think?)

https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.run_until_complete

>Return the Future’s result or raise its exception.

Reply:
Used this snippet for running unit tests in #3202:
```
import asyncio

def _run(coro):
    return asyncio.get_event_loop().run_until_complete(coro)
```
taken from:
https://blog.miguelgrinberg.com/post/unit-testing-asyncio-code

If that helps...